### DISCUSSION TITLE
Error code - 32603 when the editor logs in to the copilot

### DISCUSSION THREAD

**Discussion by AFanSKyQs** [community/community#45374] [Copilot Conversations]
Since returning home from the holiday, the editor has been logging in to copilot with error code - 32603
 I tried to connect to the school network through VPN to log in. This is really feasible, but every time I use it, I need to connect to the school network through VPN to use it. And it can only be the school network.

Recently, the VPN of the school has gone wrong, so that it has been unable to log in to the copilot.

It can be seen that the copilot may have performed IP verification. How can I solve the problem of blocking IP? Many people are not in the school during the holiday. I hope to get help. Thank you very much!
![image](https://user-images.githubusercontent.com/59954030/214678772-bcc8405b-71eb-492d-9e01-17ecc9dccda2.png)

Software: WebStorm 2022.3.0.0
Github copilot version：1.2.0.2322
No proxy or VPN connection to school

**Comment by lmlong** [comment]
I've fix this by configuring my vpn to  bypass 20.205.243.166
